runtime/doc/filetype.txt

```diff
@@ -513,40 +513,40 @@ Local mappings:
 MAN					*ft-man-plugin* *:Man* *man.vim*
 
 View manpages in Nvim. Supports highlighting, autocompletion, locales, and
-navigation. See also |find-manpage|.
+navigation. Also see |find-manpage|.
```

Comment:
I didn't even notice but "Also see" is used throughout that file, so it fits in better. I'm gonna leave it.